hazelcast/src/test/resources/log4j2.xml

```diff
@@ -36,6 +36,7 @@
         <Logger name="com.hazelcast.client.impl.operations" level="debug"/>
         <Logger name="com.hazelcast.client.impl.spi.ClientListenerService" level="trace"/>
         <Logger name="com.hazelcast.jet" level="debug"/>
+<!--        <Logger name="com.hazelcast.internal.partition.operation.MigrationRequestOperation" level="trace"/>-->
```

Comment:
minor : leftover?

Reply:
oops, just realized that it is left on-purpose for any future debugging purposes

Reply:
no leftover, put it here to debug easily when needed.